pages/SplashScreen.js

```diff
@@ -0,0 +1,14 @@
+import { StatusBar } from 'expo-status-bar';
+import React from 'react';
+import CustomButton from '../components/CustomButton.js';
+import { CustomText } from '../components/CustomText.js';
+import GeometryBackground from '../components/GeometryBackground';
+import {
+    StyledView,
+    ScrollStyledView,
+    HowToPlayContainer
+} from '../components/StyledView';
+import { BIG_MIN_HEIGHT_BUTTON } from '../components/Constants.js';
+
+
+export const SplashScreen = ({ navigation }) => {}
```

Comment:
is this file used for anything? if not, should delete this.

Reply:
I think i briefly mentioned this in the PR but I should've said it more explicitly sorry for that. This is just a skeleton that will need to be implemented soon. We also need to integrate the AppLoading component into the Splash Screen as well.